xtask/src/themelint.rs

```diff
@@ -50,6 +50,23 @@ fn get_rules() -> Vec<Require> {
         // Check for visible cursor
         Require::Existence(Rule::has_bg("ui.cursor.primary")),
         Require::Existence(Rule::has_either("ui.cursor.match")),
+        // Markup scopes
+        Require::Existence(Rule::has_either("markup.heading.1")),
+        Require::Existence(Rule::has_either("markup.heading.2")),
+        Require::Existence(Rule::has_either("markup.heading.3")),
+        Require::Existence(Rule::has_either("markup.heading.4")),
+        Require::Existence(Rule::has_either("markup.heading.5")),
+        Require::Existence(Rule::has_either("markup.heading.6")),
```

Comment:
I think it's valid for a theme to only define `markup.heading` and not `.marker` or the `.n` for headings

Reply:
Defining `markup.heading` will make all these pass, but _not_ defining it will force you to define everything.

Reply:
If anything I think the error should be on `markup.heading` instead of `.1`... which is a bit misleading.